### PR TITLE
Updated readme for review.

### PR DESCRIPTION
**Pull request by dinkarganti**
@kevinko102496 , @matt1234567890 I have added some notes for the Readme to set some context for the lab. Let's work on this document if you think that will help other members to reach out to us and engage with the group. Thanks, Dinkar

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,38 @@
-# Lab-Group
\ No newline at end of file
+# Lab-Group
+
+### Introduction
+
+[IBTTA] Working Group for blockchain technologies has been formed to understand the impact of blockchain within transportation. This group is formed to bring together experts from various parts of tolling and transportation industry to help outline applicability of blockchain (distributed-ledger, to contrast from cryptocurrency) technology.
```
Comment (matt1234567890) [inline README.md:5]:
```suggestion
The [IBTTA](https://www.ibtta.org) Blockchain Working Group for has been formed under the [Emerging Technologies Committee](https://my.ibtta.org/About-Us/Committees) to understand the impact of blockchain innovation within transportation. This group brings together experts from various disciplines in tolling and transportation to help outline the applicability of distributed ledgers, smart contracts, and tokenization.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,38 @@
...
+We love to interact with stakeholders so that we are building our mvps based on a real need. If you are interested in discussing a particular aspect of Tolling and Transportation that you would like to see addressed, please reach out to us, we will be happy to discuss and prioritize them with you.
+
+
+### Definition of MVPs - 2021
+
+Our goal for the MVPs that we would like to develop is to cover the following aspects of Tolling and Transportation:
+    - Fleet management
+    - Interoperability (IOP)
+
+We use [Monday](https://milliganpartners.monday.com/boards/1200830450/), a public company, to capture our work. Please reach out to (maintainer's name here) to be added to the board to influence our backlog and priorities.
```
Comment (matt1234567890) [inline README.md:18]:
```suggestion
We use [Monday](https://milliganpartners.monday.com/boards/1200830450/), a public company, to capture our work. Please reach out to @kevinko102496 to be added to the board to influence our backlog and priorities.
```

File: README.md [hunk 3]
```diff
@@ -1 +1,38 @@
...
+[IBTTA] Working Group for blockchain technologies has been formed to understand the impact of blockchain within transportation. This group is formed to bring together experts from various parts of tolling and transportation industry to help outline applicability of blockchain (distributed-ledger, to contrast from cryptocurrency) technology.
+
+### Interaction with Stakeholders
+
+We love to interact with stakeholders so that we are building our mvps based on a real need. If you are interested in discussing a particular aspect of Tolling and Transportation that you would like to see addressed, please reach out to us, we will be happy to discuss and prioritize them with you.
+
+
+### Definition of MVPs - 2021
+
+Our goal for the MVPs that we would like to develop is to cover the following aspects of Tolling and Transportation:
```
Comment (matt1234567890) [inline README.md:14]:
```suggestion
Our goal for the MVPs that we would like to develop is to cover the following aspects of Tolling and Transportation:

```

File: README.md [hunk 4]
```diff
@@ -1 +1,38 @@
...
+Our choice of ledger is based on some of the features listed here:
+  - Sharding is built-in
+  - Eco-friendly mining
+  - Safe-by-design smart contract language (more about this later)
+
+We began the effort by evaluating [Zilliqa](https://www.zilliqa.com), though we will be looking to port to other languages/ledgers if a suitable ledger emerges.
+
+### How to Participate in the Group
+
+Though we use the term "lab" we consider us to be a bunch of like-minded folks who like to maintain a collegial environment to exchange and bounce off ideas/code etc. If you see anything you like or don't like, please send us a note and we will be happy to discuss them and perhaps create a user story or two to memorialize them.
```
Comment (matt1234567890) [inline README.md:34]:
```suggestion
Though we use the term "lab," we consider ourselves to be a bunch of like-minded folks who like to maintain a collegial environment to exchange and bounce off ideas/code, etc. If you see anything you like or don't like, please send us a note and we will be happy to discuss them and perhaps create a user story or two to memorialize them.
```

File: README.md [hunk 5]
```diff
@@ -1 +1,38 @@
...
+
+
+### Team/Acknowledgments
```
Comment (matt1234567890) [inline README.md:37]:
```suggestion
### Team/Acknowledgments
We wish to thank these invaluable contributors and teammates:

* Dinkar Ganti, [Conduent](https://www.conduent.com)
* Kevin Ko, [Milligan Partners](https://www.milliganpartners.com)
* Matt Milligan, [Milligan Partners](https://www.milliganpartners.com)
* Devang Patel, [Kyra Solutions](https://www.kyrasolutions.com)

```

**Review by matt1234567890 — COMMENTED** [review]
@dinkarganti @dservgun Updates for your review.

File: README.md (hunk 4 above)
Comment (dinkarganti) [reply README.md:34]: @kevinko102496 i am re-adding this section for now, please feel free to edit them or link to your changes about community. Thanks.

**Comment by dinkarganti** [comment]
@matt1234567890 , @kevinko102496 I've incorporated most of the comments. Please review the pr if I may have missed anything. Thanks. Dinkar

**Review by matt1234567890 — APPROVED** [review]

**Pull request merged**